src/transformState.js

```diff
@@ -4,8 +4,33 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  actions.forEach((action) => {
+    switch (action.type) {
```

Comment:
You can use desctructurization here for action
```suggestion
  actions.forEach((action) => {
    const { type, ...} = action;
    switch (type) {
```